Packs/iDefense/Integrations/iDefense_v2/iDefense_v2.yml

```diff
@@ -0,0 +1,172 @@
+  - name: ip
+    arguments:
+    - name: ip
+      required: true
+      default: true
+      description: IP to check
+    outputs:
+    - contextPath: IP.Address
+      description: Bad IP Address found
+      type: String
+    - contextPath: IP.Malicious.Vendor
+      description: For malicious IPs, the vendor that made the decision
+      type: String
+    - contextPath: IP.Malicious.Description
+      description: For malicious IPs, the reason for the vendor to make the decision
+      type: String
+    - contextPath: DBotScore.Indicator
+      description: The indicator that was tested.
+    - contextPath: DBotScore.Type
+      description: The indicator type.
+    - contextPath: DBotScore.Vendor
+      description: The vendor used to calculate the score.
+    - contextPath: DBotScore.Score
+      description: The actual score.
+    description: Check IP reputation
+  - name: domain
+    arguments:
+    - name: domain
+      required: true
+      default: true
+      description: Domain to check
+    outputs:
+    - contextPath: Domain.Name
+      description: Bad domain found
+      type: String
+    - contextPath: Domain.Malicious.Vendor
+      description: For malicious domains, the vendor that made the decision
+      type: String
+    - contextPath: Domain.Malicious.Description
+      description: For malicious domains, the reason for the vendor to make the decision
+      type: String
+    - contextPath: DBotScore.Indicator
+      description: The indicator that was tested.
+      type: String
+    - contextPath: DBotScore.Type
+      description: The indicator type.
+      type: String
+    - contextPath: DBotScore.Vendor
+      description: The vendor used to calculate the score.
+      type: String
+    - contextPath: DBotScore.Score
+      description: The actual score.
+      type: Number
+    description: Check domain reputation
+  - name: url
+    arguments:
+    - name: url
+      required: true
+      default: true
+      description: The URL to check (must start with "http://").
+    outputs:
+    - contextPath: URL.Data
+      description: Bad URLs found
+      type: String
+    - contextPath: URL.Malicious.Vendor
+      description: For malicious URLs, the vendor that made the decision
+      type: String
+    - contextPath: URL.Malicious.Description
+      description: For malicious URLs, the reason for the vendor to make the decision
+      type: String
+    - contextPath: DBotScore.Indicator
+      description: The indicator that was tested.
+      type: String
+    - contextPath: DBotScore.Type
+      description: The indicator type.
+      type: String
+    - contextPath: DBotScore.Vendor
+      description: The vendor used to calculate the score.
+      type: String
+    - contextPath: DBotScore.Score
+      description: The actual score.
+      type: Number
+    description: Check URL reputation
+  - name: idefense-get-ioc-by-uuid
+    arguments:
+    - name: uuid
+      required: true
+      default: true
```

Comment:
though on the other hand, best not to use default for non reputation commands.
so i would do:
required: true
default: false